ui/src/plugins/com.android.PinAndroidPerfMetrics/index.ts

```diff
@@ -76,12 +76,9 @@ export default class implements PerfettoPlugin {
       },
     });
     if (metrics.length !== 0) {
-      // Add track: Android jank CUJs
-      ctx.commands.runCommand('com.android.AndroidCujs#PinJankCUJs');
-
-      // Add track: Android latency CUJs
-      ctx.commands.runCommand('com.android.AndroidCujs#PinLatencyCUJs');
-
+      const plugin = ctx.plugins.getPlugin(AndroidCujsPlugin);
+      await plugin.pinJankCujs(ctx);
+      await plugin.pinLatencyCujs(ctx);
```

Comment:
Same here with the unrelated refactor. We also need to make sure to list this dependency if we want to use the AndroidCujsPlugin